src/Room/Message.tsx

```diff
@@ -25,9 +26,20 @@ const Message: React.FC<{ message: MessageType }> = ({ message }) => {
     user.id !== message.user.id || !message.song ? (
       ''
     ) : (
-      <Button sx={{ fontSize: 1 }} onClick={pinOrUnpin}>
-        {pinned ? 'unpin' : 'pin'}
-      </Button>
+      <Box
```

Comment:
I'm not sure what I did wrong here, but the messages I pin in the chat don't seem to be showing up in the tab underneath the video.

Reply:
Even when you refresh the page?  It's not hooked up to a websocket.

Reply:
Yes, even when I refresh the page. Was it previously hooked up to a websocket? I didn't think I changed anything other than the design/text of the component...

Reply:
It wasn't -- seems to be working as expected for me:
![pinning](https://user-images.githubusercontent.com/59452077/76724225-3daf5c00-6718-11ea-9922-819d9959475d.gif)

Did you do the ol' `bin/setup` after pinning got added (to run the necessary migrations in the api project)?

Reply:
That's probably it.